packages/react-ui-components/src/SelectBox_Header/style.css

```diff
@@ -3,11 +3,35 @@
     display: flex;
 }
 
+.selectBoxHeader:after {
```

Comment:
I think this class would also affect other icons in different SelectBox configs, e.g. a search icon.
And also we don't need two lines there, only one. And it's better to render it as a separate html element, because it should be shown only when both the cross and the chevron are shown.